### ISSUE TITLE
Some verses don't return valid JSON

### ISSUE THREAD

**Issue by AnaStubbs** [seven1m/bible_api#30]
Hi,

I'm trying to interface with your API from within Second Life, an online virtual world that has its own proprietary scripting language, called LSL. LSL has some limited ability to parse JSON from an external server.

Overall, it works well, but I have found some verses that do not return valid JSON, stymieing my script. They are reliably truncated too early when requested from within Second Life, but work perfectly well in a browser (Chrome).

The verses I have found so far that reproduce this behaviour are Joshua 1:9 and Malachi 3:2.

You can find discussion of this issue at https://community.secondlife.com/forums/topic/440800-body-of-http-response-truncated-early/

I am also having trouble with the character encoding - Second Life uses UTF8, and this is what is returned in Chrome, but something else is being returned to my LSL scripts, resulting in all the quotes and apostrophes being messed up.

Is there something your server sees or fails to see in the request from Second Life that leads it to send something different from that sent to Chrome?

**Comment by seven1m** [comment]
I made a change to the app so it now returns the header`Content-Type: application/json;charset=utf-8`. Prior to that, that `charset` part was missing. While most clients I've encountered don't seem to care, I suspect that the library you're using was consuming the payload as US-ASCII or some other encoding, which could make for some weird interaction with Content-Length value.

In any case, I'm betting things are better now... give it a try!
